backend/kernelCI_app/management/commands/helpers/common.py

```diff
@@ -49,3 +50,15 @@ def get_default_tree_recipients(
             log_message(
                 f"Skipping file {filename} on loading summary files. Not a yaml file."
             )
+
+
+def setup_jinja_template(template_name: str) -> Template:
+    """Gets the template file from management/commands/templates
+    independently from where this function was called"""
+
+    base_commands_path = os.path.dirname(os.path.abspath(__file__))
+    parts = base_commands_path.split("management/commands")
+    templates_dir = os.path.join(parts[0], "management/commands", "templates")
```

Comment:
I might be wrong, but I think you can add "management/commands" to `settings.TEMPLATES["DIRS"]` to avoid this function